<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width">
    <title>Quantitative Methods in Genetics and Genomics
    </title>
    <link rel="alternate" href="https://biom262.github.io/biom262-2016/feed.xml" type="application/rss+xml" title="University of California San Diego (UCSD)
CMM262/BIOM262/BGGN237
Winter 2016">
    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Raleway:400,300,600|family=PT+Sans:400,300,600">
    <link rel="stylesheet" href="https://biom262.github.io/biom262-2016/css/normalize.css">
    <link rel="stylesheet" href="https://biom262.github.io/biom262-2016/css/skeleton.css">
    <link rel="stylesheet" href="https://biom262.github.io/biom262-2016/css/main.css">
  </head>
  <body>
    <header class="header">
      <div class="content-wrap">
        <div class="logo">
          <h1><a href="https://biom262.github.io/biom262-2016">Quantitative Methods in Genetics and Genomics</a></h1>
          <p class="description">University of California San Diego (UCSD)
CMM262/BIOM262/BGGN237
Winter 2016</p>
        </div>
      </div>
    </header>
    <div id="content">
      <div class="content-wrap"><h2 id="contact-us">Contact us</h2>
<p>Lead Instructor <a href="mailto:geneyeo@ucsd.edu" title="Gene Yeo">Dr. Gene Yeo</a><br>
Teaching Assistant <a href="mailto:obotvinn@ucsd.edu">Olga Botvinnik</a><br>
Teaching Assistant <a href="mailto:jmccorri@eng.ucsd.edu">Jamison McCorrison</a><br>
All other lecturer contact info provided below.<br></p>
<h2 id="course-description">Course Description</h2>
<p>Quantitative Methods in Genetics and Genomics is designed to teach experimental and analytical approaches in modern genetics and genomics, from experimental design through data analysis in each of several topic areas. Lecturers this year include Gene Yeo, Scott Rifkin, Bruce Hamilton, Rob Knight, Olivier Harismendy, Nicholas Schork, Aaron Chang and Hannah Carter.</p>
<p><em>This course is a requirement for students in the UCSD Genetics Training Program. 4 units.</em></p>
<h2 id="syllabus">Syllabus</h2>
<p><a href="https://gitter.im/biom262/biom262-2016?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge"><img src="https://badges.gitter.im/biom262/biom262-2016.svg" alt="Join the chat at https://gitter.im/biom262/biom262-2016"></a></p>

        <div class="syllabus"><table>
<thead>
<tr>
<th>Module Num.</th>
<th>Tues</th>
<th>Thurs</th>
<th>Lecturer</th>
<th>Topic</th>
<th></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>1</strong></td>
<td>1/5/16</td>
<td>1/7/16</td>
<td><strong><a href="http://olgabotvinnik.com">Olga Botvinnik</a>, <a href="http://yeolab.ucsd.edu/yeolab/Home.html">Gene Yeo</a></strong> (<a href="mailto:geneyeo@ucsd.edu">contact</a>)</td>
<td>TSCC Setup, UNIX, Git/Github, and Python introduction</td>
<td><a href="http://nbviewer.ipython.org/github/biom262/biom262-2016/blob/master/weeks/week01/">Lecture</a></td>
<td><a href="http://nbviewer.ipython.org/github/biom262/biom262-2016/blob/master/weeks/week01/homework.ipynb">Homework</a></td>
<td>Solution</td>
</tr>
<tr>
<td><strong>2</strong></td>
<td>1/12/16</td>
<td>1/14/16</td>
<td><strong><a href="http://labs.biology.ucsd.edu/rifkin/">Scott Rifkin</a></strong> (<a href="mailto:sarifkin@ucsd.edu">contact</a>)</td>
<td>Statistic; logic of testing hypotheses; estimation</td>
<td><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/tree/master/weeks/week02/">Lecture</a><br><a href="https://github.com/biom262/biom262-2016/blob/master/weeks/week02/conda_install_fix.md">conda install R fix</a></td>
<td><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week02/homework2.ipynb">Homework</a></td>
<td>Solution</td>
</tr>
<tr>
<td><strong>3</strong></td>
<td>1/19/16</td>
<td>1/21/16</td>
<td><strong><a href="http://cmm.ucsd.edu/Lab_Pages/hamilton/bah/">Bruce Hamilton</a></strong> (<a href="mailto:bah@ucsd.edu">contact</a>)</td>
<td>Low throughput methodology, qRT PCR</td>
<td><a href="https://github.com/biom262/biom262-2016/blob/master/weeks/week03/BIOM262_GeneExpression_2016.ppt?raw=true">Lecture</a></td>
<td><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week03/homework.ipynb">Homework</a></td>
<td>Solution</td>
</tr>
<tr>
<td><strong>4</strong></td>
<td>1/26/16</td>
<td>1/28/16</td>
<td><strong>Olga Botvinnik, Gene Yeo</strong></td>
<td>NGS Technology, RNA-seq tools and analyses, IGV</td>
<td><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week04/index.ipynb">Lecture</a> <a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week04/BIOM262-RNA%20processing.pptx?raw=true">Powerpoint Lecture</a></td>
<td><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week04/homework.ipynb">Homework</a></td>
<td>Solution</td>
</tr>
<tr>
<td><strong>5</strong></td>
<td>2/2/16</td>
<td>2/4/16</td>
<td><strong><a href="https://knightlab.ucsd.edu/">Rob Knight</a></strong> (<a href="mailto:rknight@ucsd.edu">contact</a>)</td>
<td>Microbiome</td>
<td>Lecture <a href="https://github.com/biom262/biom262-2016/blob/master/weeks/week05/evolution.py">evolution.py</a> <a href="https://github.com/biom262/biom262-2016/blob/master/weeks/week05/">Additional Materials</a></td>
<td>Homework</td>
<td>Solution</td>
</tr>
<tr>
<td><strong>6</strong></td>
<td>2/9/16</td>
<td>2/11/16</td>
<td><strong><a href="http://moores.ucsd.edu/oncogx/">Olivier Harismendy</a></strong> (<a href="mailto:oharismendy@ucsd.edu@ucsd.edu">contact</a>)</td>
<td>NGS sequencing. Analysis file standards. SNPs and SNVs in cancer. Querying public databases. Variant annotation, filtering and statistical associations.</td>
<td>Lectures: <a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week06/NGSTools_BIOM262.pptx">NGS Tools</a>, <a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week06/Practise_Day1.pptx">Practice - Day 1</a></td>
<td>Homework</td>
<td>Solution</td>
</tr>
<tr>
<td><strong>7</strong></td>
<td>2/16/16</td>
<td>2/18/16</td>
<td><strong>Aaron Chang</strong> (<a href="mailto:anchang@ucsd.edu@ucsd.edu">contact</a>)</td>
<td>Translating genomic findings into clinically relevant results. Cloud  computing resources. N of 1 case study. Visualization and interpretation in the context of precision medicine.</td>
<td>Lecture</td>
<td>Homework</td>
<td>Solution</td>
</tr>
<tr>
<td><strong>8</strong></td>
<td>2/23/16</td>
<td>2/25/16</td>
<td><strong><a href="http://www.scripps.edu/schork/Schork_Lab_Members.html">Nicholas Schork</a></strong> (<a href="mailto:nschork@ucsd.edu">contact</a>)</td>
<td>Modern Applied Population Genetics. Applied Population Genetic. Admixture Mapping.</td>
<td>Lecture</td>
<td>Homework</td>
<td>Solution</td>
</tr>
<tr>
<td><strong>9</strong></td>
<td>3/1/16</td>
<td>3/3/16</td>
<td><strong><a href="http://med.ucsd.edu/divisions/med-genetics/research/Pages/carter-lab.aspx">Hannah Carter</a></strong> (<a href="mailto:hkcarter@ucsd.edu@ucsd.edu">contact</a>)</td>
<td>Genome-wide associations studies.</td>
<td>Lecture</td>
<td>Homework</td>
<td>Solution</td>
</tr>
<tr>
<td><strong>10</strong></td>
<td>3/8/16</td>
<td>3/10/16</td>
<td><strong>Hannah Carter</strong></td>
<td>Prioritizing genetic variation. Inherited variation. Somatic variation.</td>
<td>Lecture</td>
<td>Homework</td>
<td>Solution</td>
</tr>
</tbody>
</table>

        </div><h2 id="tips-and-tricks">Tips and tricks</h2>
<ul>
<li><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week01/homework.ipynb#FAQ-+-Tips-and-Tricks">FAQ from homework 1</a> (ssh, logging in to TSCC&lt; starting up a notebook, file navigation)</li>
<li><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/git_essentials.ipynb">Git essentials</a></li>
<li><a href="http://nbviewer.jupyter.org/github/biom262/biom262-2016/blob/master/weeks/week04/fixing_unreadable_notebook_error_merge_conflicts.ipynb">Fixing unreadable notebook error</a></li>
<li><a href="http://staffwww.fullcoll.edu/sedwards/Nano/NanoKeyboardCommands.html">Nano keyboard shortcuts</a></li>
</ul>
<h2 id="class-location">Class Location</h2>
<p>All lectures throughout the quarter will be held in the <a href="https://www.google.com/maps/place/Sanford+Consortium+for+Regenerative+Medicine/@32.8895137,-117.2462225,17z/data=!4m6!1m3!3m2!1s0x80dc06934d4eabd9:0x97741ae2ff2bd85a!2sSanford+Consortium+for+Regenerative+Medicine!3m1!1s0x80dc06934d4eabd9:0x97741ae2ff2bd85a">Sanford Consortium for Regenerative Medicine (SCRM)</a> building.</p>
<ul>
<li><strong>For the first week of the quarter (1/5 and 1/7)</strong>, the class will be held in the second floor auditorium. This is the large auditorium besides the Bella Vista Cafe on the second floor.</li>
<li><strong>For the remainder of the quarter</strong>, all lectures will be held in the first floor conference room (#1013).</li>
<li>All rooms have been reserved from 8:45 am to 12:15 pm.</li>
<li><em>Class will start promptly at 9:00 am.</em></li>
</ul>
<h2 id="final">Final</h2>
<p><strong>Take home exam due Monday March 14, 2016</strong></p>

      </div>
    </div>
    <footer>
      <div class="content-wrap">
        <section class="about"><p><a href="https://github.com/biom262/biom262-2016/">Class Github repo</a></p>
<p><a href="https://github.com/biom262/biom262-2016/tree/master/website">Github repo that creates this webpage</a></p>
<p><a href="https://github.com/biom262/biom262-2016/blob/master/.travis.yml">Travis-CI Build script</a></p>
<p><a href="https://travis-ci.org/biom262/biom262-2016"><img src="https://travis-ci.org/biom262/biom262-2016.svg?branch=master" alt="Build Status"></a></p>

        </section>
        <section class="copy">
          <p>&copy; 2016 BIOM262 Staff &mdash; powered by&nbsp;<a href="https://github.com/jnordberg/wintersmith">Wintersmith</a>
          </p>
        </section>
      </div>
    </footer>
  </body>
</html>